docs/glsl-fuzz-walkthrough.md

```diff
@@ -570,6 +594,8 @@ was enough to cause the wrong image to be rendered.
 
 ## Queuing a no image reduction
 
+> Reductions for the case where no SSBO is produced by a compute shader cannot be launched via the Web UI.  [See how to launch them via the command line](#Performing-a-"no-image"-reduction-on-a-fragment-or-compute-shader).
```

Comment:
Are the double quotes (") OK in an internal link? spotted in `(#Performing-a-"no-image"-reduction-on-a-fragment-or-compute-shader)`

Reply:
VS code was OK with this, but it seems that it doesn't work on GitHub.  I've removed the quotes.